BedrockCommand.h

```diff
@@ -118,6 +118,9 @@ class BedrockCommand : public SQLiteCommand {
     // Returns true if all of the httpsRequests for this command are complete (or if it has none).
     bool areHttpsRequestsComplete() const;
 
+    // Returns a map with all the parameters that can be used to log information about the command being run.
+    map<string,string> getCommandLogParams();
```

Comment:
Needs to be virtual if you want to override in auth.

Reply:
We won't overwrite it in Auth since we're logging everything from the request, that's why it's not virtual.

Reply:
Then I don't see why we need it. I thought we were adding it so that auth can return a different list of params.

Also:

<img width="803" alt="Screenshot 2024-11-26 at 13 03 13" src="https://github.com/user-attachments/assets/51341f36-fbfb-4a44-8145-f977aa269b9e">

Reply:
It was at first, but after adding everything in the request we don't need auth to do that anymore. Let me fix the PR description.